examples/release_highlights/plot_release_highlights_1_0.py

```diff
@@ -0,0 +1,184 @@
+# flake8: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.0
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.0! The library has
+been stable for quite some time, releasing version 1.0 is recognizing that and
+signalling it to our users. The release does not include any breaking changes,
+and as usual, and we do our best to follow a two-release deprecation cycles for
+any breaking changes.
+
+This release includes many bug fixes and improvements, as well as some new key
+features. We detail below a few of the major features of this release. **For an
+exhaustive list of all the changes**, please refer to the :ref:`release notes
+<changes_1_0>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+"""
+
+##############################################################################
+# Keyword and positional arguments
+# ---------------------------------------------------------
+# The scikit-learn API exposes many functions and methods which have many input
+# parameters. For example, before this release, one could instantiate a
+# :class:`~ensemble.HistGradientBoostingRegressor` as::
+#
+#         HistGradientBoostingRegressor("squared_error", 0.1, 100, 31, None,
+#             20, 0.0, 255, None, None, False, "auto", "loss", 0.1, 10, 1e-7,
```

Comment:
```suggestion
#             20, 0.0, 255, None, None, False, "auto", "squared_error", 0.1, 10, 1e-7,
```

Reply:
This is setting the "scoring" attribute where `'squared_error`' is an option.

Reply:
why does it matter? This is setting all with their default values

Reply:
My bad, it doesn't matter.